packages/validator/test/cli-validator/tests/expected-output.test.js

```diff
@@ -377,12 +377,12 @@ describe('test expected output - OpenAPI 3', function() {
 
   it('should include the path to the component (if it exists) when in verbose mode', async function() {
     const program = {};
-    program.args = ['./test/cli-validator/mock-files/oas3/testoneof.yaml'];
+    program.args = ['./test/cli-validator/mock-files/oas3/component-path-example.yaml'];
```

Comment:
Updating the code resolved all the validations present in `testoneof`, so component paths are not relevant anymore. I wrote a new API definition that includes unresolved validations from our own rules to test the "component paths" feature.